mkdocs.yml

```diff
@@ -217,7 +218,7 @@ nav:
           - MDIMs and Explorers: "guides/data-work/mdims.md"
           - Export data: "guides/data-work/export-data.md"
           - Access archived data: "guides/data-work/access-archived-data.md"
-          - GSheets for OMMs: "guides/data-work/create-gsheets-omm.md"
+          - Create a public GSheet for a variable: "guides/data-work/create-gsheet.md"
```

Comment:
Sorry to insist here, maybe this one is too long.

I'd use "indicator" instead of "variable", to be aligned with our language.

Maybe something like: "Publish indicator metadata with GSheet".

I find it also clarifies why we would want to use Gsheets at all.

Reply:
Sure I set it as: 'Publish indicator and metadata with GSheet', still a bit long but I think it's important to note that we're publishing the indicator values as well as the metadata.